ivadomed/training.py

```diff
@@ -316,10 +359,17 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
 
     # Save best model in output path
     if resume_path.is_file():
-        state = torch.load(resume_path)
+        # loading state changes based on CPU or DDP
+        if local_rank == -1 and torch.cuda.device_count() <= 1:
```

Comment:
This condition could be replaced with a variable, to avoid repetition of the same statement in this file

```
no_ddp = local_rank == -1 and torch.cuda.device_count() <= 1
if no_ddp:
    blah()

```